agent/src/main/java/kanela/agent/api/instrumentation/listener/InstrumentationRegistryListener.java

```diff
@@ -35,12 +36,16 @@
 
     private Map<String, Map<TypeTransformation, List<TypeDescription>>> moduleTransformers = HashMap.empty();
     private Map<String, KanelaConfiguration.ModuleConfiguration> modulesConfiguration = HashMap.empty();
+    private Map<String, String> modulesVersion = HashMap.empty();
     private Map<String, List<Throwable>> errors = HashMap.empty();
 
     public Map<String, Map<TypeTransformation, List<TypeDescription>>> getModuleTransformers() {
         return moduleTransformers;
     }
 
+    public String moduleVersion(String moduleKey) {
+        return modulesVersion.get(moduleKey).getOrNull();
```

Comment:
why return `null`? why not an `Option`?

Reply:
done, move the `getOrNull` to the class who write the info json, which need the null value